# There is small and cute import problem when OpenGL is imported by MCEdit
#
# Log:
# [INFO][formathandler.py:46]:Unable to load registered array format handler vbo:
# Traceback (most recent call last):
#   File "/usr/lib/python2.7/site-packages/OpenGL/arrays/formathandler.py", line 35, in loadPlugin
#     plugin_class = entrypoint.load()
#   File "/usr/lib/python2.7/site-packages/OpenGL/plugins.py", line 14, in load
#     return importByName( self.import_path )
#   File "/usr/lib/python2.7/site-packages/OpenGL/plugins.py", line 28, in importByName
#     module = __import__( ".".join(moduleName), {}, {}, moduleName)
#   File "/usr/lib/python2.7/site-packages/OpenGL/arrays/vbo.py", line 33, in <module>
#     from OpenGL import GL
# ImportError: cannot import name GL

--- mcedit.py.orig	2012-11-18 02:28:59.315992784 +0100
+++ mcedit.py	2012-11-18 02:29:40.380719147 +0100
@@ -5,6 +5,7 @@
 
 Startup, main menu, keyboard configuration, automatic updating.
 """
+from OpenGL import GL
 import OpenGL
 import sys
 import os
